src/pages/TopUp/index.tsx

```diff
@@ -130,13 +127,20 @@ const _TopUpPage: React.FC<Props> = () => {
                 `${BEACONCHAIN_URL}/api/v1/validator/${pubKeysCommaDelineated}`
               )
                 .then(r => r.json())
-                .then(({ data }: { data: BeaconChainValidator[] }) => {
-                  if (data.length === 0) {
-                    setShowDepositVerificationWarning(true);
+                .then(
```

Comment:
@hwwhww Looks like this is actually the only response that has the Array vs Object caveat... not the `eth1` fetch above. This code could use a little modernizing as well, so I can try to clean this up and push the results if you'd like.